app/src/main/res/layout/add_profile_activity.xml

```diff
@@ -85,7 +85,8 @@
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@+id/add_profile_activity_required_heading_text_view"
-            profile:error="@{viewModel.nameErrorMsg}" />
+            profile:error="@{viewModel.nameErrorMsg}"
+            profile:singleLine="@{true}" />
```

Comment:
This same change should be done in `land/add_profile_activity.xml` too.

Reply:
Done